steam-redirector/Makefile

```diff
@@ -10,5 +10,8 @@ main: main.c unix_utils.c
 	$(GNU_C) -o main unix_utils.c main.c
 
 main.exe: main.c win32_utils.c
+	$(WIN_C) -mwindows -municode -static -static-libgcc -Bstatic -lpthread -o main.exe win32_utils.c main.c
+
+main_debug.exe: main.c win32_utils.c
 	$(WIN_C) -municode -static -static-libgcc -Bstatic -lpthread -o main.exe win32_utils.c main.c
```

Comment:
While the target is defined as `main_debug.exe`, it is creating a file named `main.exe`. This would break the system `make` uses to identify whether files need to be re-built or not, eg.:
1. User builds `main.exe` by executing `make main_debug.exe`;
2. `make` creates a `main.exe` file;
3. The user decides it now wants a non-debug version of the binary and executes `make main.exe`;
4. `make` will not create `main.exe` because it was already built in step 1 and neither `main.c` nor `win32_utils.c` (main.exe dependencies, defined after the colon) have changed since `main.exe` was built;

In general, you want to have target names which match the name of the file they generate. When you want to have very specific targets with don't refer to files (install, clean, etc), you usually mark them as [phony targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) so make knows they don't refer to files. While it is possible to mark some targets as phony to prevent the aforementioned problem, it would be a highly unusual solution and cause make to rebuild the targets even when not necessary.

You can just have the target compile a `main_debug.exe` file.

Reply:
Done in https://github.com/rockerbacon/modorganizer2-linux-installer/pull/413/commits/57f7b688a2071230ba0fb3683c22bdc1ff0125b4

Thanks for your advice, I'm pretty new to contributing on github and I have to admit I didn't gave it as much thoughts as I should have.